es6/namespace.js

```diff
@@ -84,6 +84,8 @@ class SimpleNamespace extends Namespace {
     this.defaultExperimentClass = DefaultExperiment
     this._inExperiment = false;
 
+    this._onlyLogExposureIfPresent = true;
```

Comment:
It seems that if an experiment does not explicitly define a (randomized) treatment assignment, it should never be exposure logged -- so this value should aways be true.